frontends/web/src/routes/device/bitbox01/bitbox01.jsx

```diff
@@ -101,9 +103,9 @@ class Device extends Component {
     if (this.state.deviceRegistered) {
       apiGet('devices/' + this.props.deviceID + '/status').then(deviceStatus => {
         if (['seeded', 'initialized'].includes(deviceStatus)) {
-          setSidebarStatus('');
+          this.context.setSidebarStatus('');
         } else {
-          setSidebarStatus('forceHidden');
+          this.context.setSidebarStatus('forceHidden');
```

Comment:
I noticed a small bug which seems to be present since a long time (also in v4.39.0)

1) open BBApp
2) insert unseeded BB01
3) unplug
4) hamburger is gone

no problem with BB02. I don't think many people use BB01 anymore and run into this issue, else they can just reopen BBApp.